modules/virt-using-wasp-agent-to-configure-higher-vm-workload-density.adoc

```diff
@@ -18,13 +17,98 @@ For descriptions of QoS classes, see link:https://kubernetes.io/docs/tasks/confi
 
 .Prerequisites
 
-* The `oc` tool is available.
-* You are logged into the cluster with the cluster-admin role.
-* A memory over-commit ratio is defined.
+* You have installed the OpenShift CLI (`oc`).
+* You are logged into the cluster with the `cluster-admin` role.
+* A memory overcommit ratio is defined.
 * The node belongs to a worker pool.
 
+[NOTE]
+====
+The `wasp-agent` component deploys an OCI hook to enable swap usage for containers on the node level. The low-level nature requires the `DaemonSet` object to be privileged.
+====
+
 .Procedure
 
+. Configure the `kubelet` service to permit swap:
+.. Create or edit a `KubeletConfig` file with the parameters shown in the following example:
++
+.Example of a `KubeletConfig` file
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: custom-config
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: ''  # MCP
+      #machine.openshift.io/cluster-api-machine-role: worker # machine
+      #node-role.kubernetes.io/worker: '' # node
+  kubeletConfig:
+    failSwapOn: false
+----
+
+.. Wait for the worker nodes to sync with the new configuration by running the following command:
```

Comment:
Is the step here to wait, or is it to sync with the new configuration? I'm wondering if something like this would be accurate:
`Run the following command to sync the worker nodes with the new configuration:`

Also, I believe the OCP docs started to move away from using "worker node" in favor of "compute node". I see that the actual command includes the term `worker`, so I can see why you might still use that in the related step, though.

Reply:
I think the command tells the `oc` CLI to wait until the `worker` mcp has the `Updated=True` condition, so I'm going to leave it as-is. 

Good point about compute/worker nodes; I will look into that for the future.